### DISCUSSION TITLE
🔗 Elementor Top Bar Missing Viewport Size (Duplicate of #22432)

### DISCUSSION THREAD

**Discussion by maurocarreraok** [elementor/elementor#24404] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

On the new Elementor Top Bar, it would be nice to have the current viewport size. Previously, we had the flexibility to adjust the 'viewport' to specific sizes by dragging the menu to the right or even adjusting the screen to a specific size. It would be great to have that functionality back.

I'm referring to this:
<img width='195' alt='Screenshot 2023-11-25 at 13 18 48' src='https://github.com/elementor/elementor/assets/68023943/e689053b-3f39-4647-9605-288e01b9ffed'>

### Describe the solution you'd like

Put it back :)

**Comment by nicholaszein** [comment]
Hey, @maurocarreraok!

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #22432

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards
